docs/changelog-fragments/14.doc.rst

```diff
@@ -0,0 +1,2 @@
+Update requirements file to replace
```

Comment:
This is not a doc change. It's rather internal.

Reply:
not necessarily ...it is also a user/dev facing change.

Reply:
End-users don't use this file. Only the core devs of this project do.

Reply:
FWIW this is not a documentation change, it does not edit any docs.

Reply:
Not only core dev anyone who wants to contribute to the project can use the file, so it should be in changelog. I will add it as misc change